docs/content/migration/helm-chart.md

```diff
@@ -20,3 +20,8 @@ and `mesh.logLevel` options to configure the logging level for the controller an
 
 The `smi.enable` option has been deprecated and removed. You should use the new and backward compatible ACL mode 
 option as described in the [documentation](../install.md#access-control-list). 
+
+### Default Mode
```

Comment:
This description should be in a section for the next Helm chart version. 
Something like `v2.0 to v2.1`?